log-manager/src/test/java/io/airlift/log/TestRollingFileMessageOutput.java

```diff
@@ -47,16 +47,87 @@
 import static org.testng.Assert.assertTrue;
 
 @Test(timeOut = 5 * 60 * 1000)
-public class TestRollingFileHandler
+public class TestRollingFileMessageOutput
 {
+    private static void assertCompression(Path masterFile, BufferedHandler handler, String message, int expectedFileCount, int expectedLineCount, int expectedCompressedSize)
```

Comment:
These should be at the bottom of the file not the top... all static helpers at the bottom

Reply:
I think intellij reordered these, sorry.